hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1681,16 +1739,16 @@ bool MDMParser::getDataUsage(MDM_DataUsage &data)
             data.rx_total = _data_usage.rx_total;
         }
     }
-    UNLOCK();
     return ok;
 }
 
 bool MDMParser::getCellularGlobalIdentity(CellularGlobalIdentity& cgi_) {
     LOCK();
 
     // Ensure modem is responsive
-    if (!_atOk())
+    if (!_checkModem()) {
         goto failure;
+    }
```

Comment:
More single line statements below to spruce up with braces, thanks! 😸 
```
if (RESP_OK != waitFinalResp(nullptr, nullptr, COPS_TIMEOUT))
        goto failure;
```

```
if (RESP_OK != waitFinalResp(_cbCOPS, &_net, COPS_TIMEOUT))
        goto failure;
```

```
if (cgi_.size < sizeof(_net.cgi))
            goto failure;
```

Reply:
I always try to clean up at least within the scope of the function I'm currently making changes in.  If we wait to clean up later, it will likely never happen any time soon.